apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -195,6 +195,7 @@ function EventTypeSingleLayout({
       backPath="/event-types"
       title={eventType.title + " | " + t("event_type")}
       heading={eventType.title}
+      showHeading
```

Comment:
I am confused why is there a showHeading prop? I would always want to show the heading.

Edit: https://github.com/calcom/cal.com/issues/5420 I found out that this was requirement to hide the heading in this case which is why hiding in mobile was implemented.